gubernator/filters.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright 2016 Google Inc.
```

Comment:
This should probably line up with https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.py.txt